### DISCUSSION TITLE
[Q&A] Support policy for Fluentd

### DISCUSSION THREAD

**Discussion by shazib-summar** [fluent/fluentd#4549] [Q&A] [answered]
### What is a problem?

Hello. Im interested to know about the end of life policy for Fluentd. I came across the following links.
- [Scheduled support lifecycle announcement about Fluent Package](https://www.fluentd.org/blog/fluent-package-scheduled-lifecycle) blog post
- [New Release Cycle for td-agent (Fluentd LTS)](https://github.com/fluent/fluentd/discussions/3860) GitHub ticket

From these links we can learn that the support policy for fluent-package is N, meaning the latest minor release is supported at any time. Is this understanding correct? What about Fluentd itself, does that follow the same support policy as fluent-package

Thanks in advance.

### Environment

```markdown
N/A
```

**Comment by kenhys** [comment, marked as answer]
Currently fluentd v1.17.x and v1.16.x are supported.

ref.
https://github.com/fluent/fluentd/blob/master/SECURITY.md

**Reply by shazib-summar** [reply]
thank you so much. this is what i needed.
